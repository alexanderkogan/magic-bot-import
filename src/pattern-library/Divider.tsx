import * as React from 'react'

export default () => <hr className="obc_divider" />
