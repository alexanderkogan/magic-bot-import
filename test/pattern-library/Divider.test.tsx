import * as React from 'react'

import Divider from '../../src/pattern-library/Divider'
import { mount } from '../test-utils'

describe('Content', () => {
    it('renders', () => {
        const element = mount(<Divider />)
        expect(element).toMatchSnapshot()
    })
})
