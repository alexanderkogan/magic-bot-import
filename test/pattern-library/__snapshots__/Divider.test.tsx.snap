// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content renders 1`] = `
<Component>
  <hr
    className="obc_divider"
  />
</Component>
`;
